Makefile

```
@@ -201,28 +208,37 @@ generate: ##@other Regenerate assets and other auto-generated stuff
 	$(shell cd ./services/shhext/chat && exec protoc --go_out=. ./*.proto)
 
 prepare-release: clean-release
-	mkdir -p $(RELEASE_DIRECTORY)
-	mv build/bin/statusgo.aar $(RELEASE_DIRECTORY)/status-go-android.aar
+	mkdir -p $(RELEASE_DIR)
+	mv build/bin/statusgo.aar $(RELEASE_DIR)/status-go-android.aar
 	zip -r build/bin/Statusgo.framework.zip build/bin/Statusgo.framework
-	mv build/bin/Statusgo.framework.zip $(RELEASE_DIRECTORY)/status-go-ios.zip
-	zip -r $(RELEASE_DIRECTORY)/status-go-desktop.zip . -x *.git*
+	mv build/bin/Statusgo.framework.zip $(RELEASE_DIR)/status-go-ios.zip
+	zip -r $(RELEASE_DIR)/status-go-desktop.zip . -x *.git*
 	${MAKE} clean
 
 clean-release:
-	rm -rf $(RELEASE_DIRECTORY)
+	rm -rf $(RELEASE_DIR)
 
 release:
-	@read -p "Are you sure you want to create a new GitHub $(shell if [ $(PRE_RELEASE) = "0" ] ; then echo release; else echo pre-release ; fi) against $(RELEASE_BRANCH) branch? (y/n): " REPLY; \
+	@read -p "Are you sure you want to create a new GitHub $(RELEASE_TYPE} against $(RELEASE_BRANCH) branch? (y/n): " REPLY; \
 	if [ $$REPLY = "y" ]; then \
 		latest_tag=$$(git describe --tags `git rev-list --tags --max-count=1`); \
 		comparison="$$latest_tag..HEAD"; \
 		if [ -z "$$latest_tag" ]; then comparison=""; fi; \
 		changelog=$$(git log $$comparison --oneline --no-merges --format="* %h %s"); \
-	    github-release $(shell if [ $(PRE_RELEASE) != "0" ] ; then echo "-prerelease" ; fi) "status-im/status-go" "v$(RELEASE_TAG)" "$(RELEASE_BRANCH)" "$(changelog)" "$(RELEASE_DIRECTORY)/*" ; \
+		github-release \
+			$(shell if [ $(PRE_RELEASE) != "0" ] ; then echo "-prerelease" ; fi) \
+			"status-im/status-go" \
+			"v$(RELEASE_TAG)" \
+			"$(RELEASE_BRANCH)" \
+			"$(changelog)" \
+			"$(RELEASE_DIR)/*" ; \
 	else \
 	    echo "Aborting." && exit 1; \
 	fi
 
+xgo-install:
+	go get github.com/karalabe/xgo
```

```suggestion
	go get -u github.com/karalabe/xgo
```

Without `-u`, it will not update the currently installed version, potentially using some pretty old one. With `-u` it will update it, but in the CI you get non-determinism. It shouldn't be a big deal because xgo rarely changes.

---

Good point!